Gemfile.lock

```diff
@@ -1036,4 +1036,4 @@ DEPENDENCIES
   webmock
 
 BUNDLED WITH
-   2.4.19
+   2.5.23
```

Comment:
Please just make sure this is intended and not a mix of your `develop` branch and the Dockerfiles consolidation branch.

Reply:
OK, thanks!

Reply:
Since Brakeman complained, I re-ran specifying the bundler version:

```
docker compose run api bash -c "bundle update pender_client --bundler 2.4.19"
```